test/policy-collection/policy_comp_operator_test.go

```diff
@@ -80,13 +79,18 @@ var _ = Describe("RHACM4K-2222 GRC: [P1][Sev1][policy-grc] Test compliance opera
 	const compPolicyName = "policy-comp-operator"
 	const compE8ScanPolicyURL = "https://raw.githubusercontent.com/open-cluster-management/policy-collection/main/stable/CM-Configuration-Management/policy-compliance-operator-e8-scan.yaml"
 	const compE8ScanPolicyName = "policy-e8-scan"
+	var getComplianceState func() interface{}
+
 	BeforeEach(func() {
 		if !isOCP46andAbove() {
 			Skip("Skipping as compliance operator is only supported on OCP 4.6 and above")
 		}
 		if !canCreateOpenshiftNamespaces() {
 			Skip("Skipping as compliance operator requires the ability to create the openshift-compliance namespace")
 		}
+
+		// Assign this here to avoid using nil pointers as arguments
+		getComplianceState = common.GetComplianceState(clientHubDynamic, userNamespace, compPolicyName, clusterNamespace)
```

Comment:
Interesting - I wasn't expecting this. What would be a nil pointer if you don't do this?

Reply:
`clientHubDynamic` et al would be nil if you tried to define this outside of the suite, is that right? Doesn't matter much, just curious.

Reply:
For whatever reason, `userNamespace` ended up being a nil pointer. I think it's the timing issue you mentioned in #137 about the parsing of CLI arguments by ginkgo.